README.md

```diff
@@ -150,7 +150,7 @@ Each day I take one subject from the list below, read it cover to cover, take no
 - [x] [The Machine Learning Mastery Method](http://machinelearningmastery.com/machine-learning-mastery-method/)
 - [x] [Machine Learning for Programmers](http://machinelearningmastery.com/machine-learning-for-programmers/)
 - [x] [Applied Machine Learning with Machine Learning Mastery](http://machinelearningmastery.com/start-here/)
-- [ ] [Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
+- [ ] [(In Progress) Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
```

Comment:
My lasso results were bunk from yesterday. Interestingly, LogisticRegression with Kfold 10 only returns ~25% accuracy with the abalone data set, but much higher accuracy with the diabetes data set here: "https://goo.gl/vhm1eU". I think there are far fewer records.

Also notices that LeaveOneOut cross validation (which is the same as KFold(n_folds=1)) takes a bit longer to run.